packages/plugin-doc-coverage/src/lib/doc-coverage-plugin.unit.test.ts

```diff
@@ -0,0 +1,37 @@
+import { describe, expect, it } from 'vitest';
+import type { RunnerConfig } from '@code-pushup/models';
+import { PLUGIN_SLUG } from './constants.js';
+import {
+  PLUGIN_DESCRIPTION,
+  PLUGIN_DOCS_URL,
+  PLUGIN_TITLE,
+  docCoveragePlugin,
+} from './doc-coverage-plugin.js';
+
+vi.mock('./runner/index.ts', () => ({
+  createRunnerConfig: vi.fn().mockReturnValue({
+    command: 'node',
+    outputFile: 'runner-output.json',
+  } satisfies RunnerConfig),
+}));
+
+describe('docCoveragePlugin', () => {
```

Comment:
The following tests could get added:
- should throw for invalid plugin options
- should filter groups
- should filter audits
- should forward options to runner function